.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,71 @@
+# For most projects, this workflow file will not need changing; you simply need
```

Comment:
We don't need the ASF license header for this file? As I saw the header is added for other workflow files.

Reply:
Updated.